functions/Copy-SqlSsisCatalog.ps1

```diff
@@ -0,0 +1,472 @@
+
+Function Copy-SqlSsisCatalog
+{
+<#
+.SYNOPSIS 
+Copy-SqlSsisCatalog migrates Folders, SSIS projects, and environments from one SQL Server to another. 
+.DESCRIPTION
+By default, all folders, projects, and environments are copied. 
+The -Project parameter can be specified to copy only one project, if desired.
+The parameters get more granular from the Folder level.  i.e. specifying folder will only deploy projects/environments from within that folder.
+.PARAMETER Source
+Source SQL Server.You must have sysadmin access and server version must be SQL Server version 2012 or greater.
+.PARAMETER Destination
+Destination Sql Server. You must have sysadmin access and server version must be SQL Server version 2012 or greater.
+.PARAMETER Project
+Specify a source Project name.
+.PARAMETER Folder
+Specify a source folder name.
+.PARAMETER Environment
+Specify an environment to copy over.
+.PARAMETER CreateCatalogPassword
+If a destination SSISDB catalog needs to be created, specify this secure string parameter to skip password prompts.
+.NOTES 
+Original Author: Phil Schwartz
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+.LINK
+https://dbatools.io/functions/Copy-SqlSsisCatalog
```

Comment:
The shortlink is valid -- this hsould be https://dbatools.io/Copy-SqlSsisCatalog